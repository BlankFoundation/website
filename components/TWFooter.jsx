import {
  TWFlex
} from '.'

const TWFooter = ({ className, children }) => {
  return (
    <div className={`${className} p-3 bg-gray-900 text-white text-xs leading-10`}>
      <TWFlex className='sm:px-24'>
        {children}
      </TWFlex>
    </div>
  );
}

TWFooter.defaultProps = {
  children: ['child1', 'child2', 'child3'].map(
    (child, index) => <div key={'child-' + index}>{child}</div>
  ),
  className: ""
}

export default TWFooter;