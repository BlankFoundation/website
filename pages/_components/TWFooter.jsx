import {
  TWFlex
} from '.'

const TWFooter = ({ className, children }) => {
  return (
    <div className={`${className} p-6 bg-gray-900 text-white text-xs leading-10`}>
      <TWFlex className='px-24'>
        {children}
      </TWFlex>
    </div>
  );
}

TWFooter.defaultProps = {
  children: ['child1', 'child2', 'child3'].map(
    (child, index) => <div key={'child-' + index}>{child}</div>
  ),
  className: ""
}

export default TWFooter;