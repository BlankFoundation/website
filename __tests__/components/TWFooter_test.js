import React from 'react';
import TestRenderer from 'react-test-renderer';
import TWFooter from '../../components/TWFooter';

describe("TWFooter", () => {
  it('renders correctly', () => {
    const rendered = TestRenderer.create(<TWFooter />).toJSON();
    expect(rendered).toMatchSnapshot()
  });
});