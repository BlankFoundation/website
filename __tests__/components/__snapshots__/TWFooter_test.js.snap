// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TWFooter renders correctly 1`] = `
<div
  className=" p-3 bg-gray-900 text-white text-xs leading-10"
>
  <div
    className="sm:px-24 flex flex-row justify-between"
  >
    <div>
      child1
    </div>
    <div>
      child2
    </div>
    <div>
      child3
    </div>
  </div>
</div>
`;
